www/include/configuration/configObject/service/formService.php

```diff
@@ -62,10 +65,12 @@ function myDecodeService($arg)
 
 if (!$centreon->user->admin) {
     if ($service_id) {
-        $checkres = $pearDB->query("SELECT service_id
-                                        FROM $acldbname.centreon_acl
-                                        WHERE service_id = " . $pearDB->escape($service_id) . "
-                                        AND group_id IN (" . $acl->getAccessGroupsString() . ")");
+        $checkres = $pearDB->query(
+            "SELECT service_id
+            FROM {$acldbname}.centreon_acl
+            WHERE service_id = " . ((int) $service_id)
+            . "  AND group_id IN (" . $acl->getAccessGroupsString() . ")"
+        );
```

Comment:
Same question than https://github.com/centreon/centreon/pull/9298/files#r543139705

Reply:
There is no PDO in PHP 5.4